knative-operator/deploy/resources/knativekafka/channel/1-channel-consolidated.yaml

```diff
@@ -736,6 +770,9 @@ spec:
                     namespace:
                       description: 'Namespace of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/ This is optional field, it gets defaulted to the object holding it if left out.'
                       type: string
+                deadLetterSinkUri:
```

Comment:
related this PR from upstream 

https://github.com/knative-sandbox/eventing-kafka/pull/1057#issuecomment-1012836803